warp_pipe.go

```diff
@@ -163,6 +164,25 @@ func (w *WarpPipe) Close() error {
 	return nil
 }
 
+// IsLatestChangeSet returns true if the id argument matches that of the last record in the changeset table.
+func (w *WarpPipe) IsLatestChangeSet(id int64) (bool, error) {
```

Comment:
This implementation will only work for "audit" mode (where we are using `LISTEN/NOTIFY`) and will not work for logical replication mode. So we should perhaps make WP panic if you are running logical replication mode and have enabled the flag for auto-shutdown-on-complete-sync behavior. (see: commend above)

Reply:
Yes. This is something I need to fix up. I think it needs to be moved to the listener interface? Was going to return a not supported error for LR? Unless you know of a way to determine this?